### ISSUE TITLE
Documentation of the architecture

### ISSUE THREAD

**Issue by debuglevel** [cyborgbackup/cyborgbackup#75]
I read the current docs, but I did not really understand how CyBorgBackup works in a setup of N repositories/destinations and M nodes/sources:
* Is CyBorgBackup installed on each node and manages when and how it backups?
* Is CyBorgBackup installed on the destination host containing the repositories?
* Is CyBorgBackup installed on a separate host?
* Does it fully control everything (e.g. create SSH keys, distribute it to the nodes, configure borg) or do I have to do stuff myself?
* Can it be used as an additional "just get me some information without battling the CLI"-web interface in combination with existing `borg` and `borgmatic` stuff?

I think it would be nice to have some kind of walkthrough :-). Because the screenshots and the general idea of having some kind of UI are nice, but in the current state of the docs I would not consider to switch from my `borgmatic` setup.

**Comment by GaetanF** [comment]
Hello,

CyBorgBackup can be installed on a separate host. This host can also be used as repositories and/or as client for CyBorgBackup.

He will launch borg backup command on each client depending of the policy choosen using SSH connection. Depending of the CPU core available on the host, he can launch multiple backup in parallel depending of number of repositories configured and without run multiple job on the same host or the same repository.

Actually, he doesn't configure the SSH requirements. You need to create an SSH Key (or use the setting generation button) and deploy them to the different hosts (repositories and clients) to permit CyBorgBackup to connect to them without password.

CyBorgBackup will deploy automatically Borg on clients and repository hosts. He initialize the repository if the folder does not match to a Borg Repository.

CyBorgBackup cannot discover backup already done using borg manually. He only knows his own jobs.
